### DISCUSSION TITLE
How to print walls first, then overhang?

### DISCUSSION THREAD

**Discussion by pputnik** [OrcaSlicer/OrcaSlicer#5103] [Q&A]
Hello All,
The idea/goal: when there's a wall, an overhang in the same layer can stick its side to the wall instead of just "hang over". For some reason, the slicer decides to pring overhang first. On the screenshot below there are two identical elements so on the left the layer is completed, on the right - not yet.  

I've tried "Walls printing order" - didn't help and the logic says it shouldn't. Also, I tried to switch to Arachne - it helped in one place but not this one.

Also, in this and many other cases, I believe we'll have a much better quality if overhangs are printed not along the gap but across: it makes the distance to hang shorter. Is there an option to do it this way?
<img width="235" alt="1" src="https://github.com/SoftFever/OrcaSlicer/assets/10958522/01d50afa-6c3a-4b67-b23e-31100747411c">

Thank you.

**Comment by vgdh** [comment]
year bridge direction detection works wrong some times.
